packages/okta-vue/README.md

```diff
@@ -0,0 +1,223 @@
+
+### Show Login and Logout Buttons
+In the relevant location in your application, you will want to provide `Login` and `Logout` buttons for the user. You can show/hide the correct button by using the `$auth.isAuthenticated()` method. For example:
+
+```typescript
+// src/App.vue
+
+<template>
+  <div id="app">
+    <router-link to="/" tag="button" id='home-button'> Home </router-link>
+    <button v-if='authenticated' v-on:click='logout' id='logout-button'> Logout </button>
+    <button v-else v-on:click='$auth.loginRedirect' id='login-button'> Login </button>
+    <router-view/>
+  </div>
+</template>
+
+<script>
+
+export default {
+  name: 'app',
+  data: function () {
+    return { authenticated: false }
+  },
+  created () { this.isAuthenticated() },
+  watch: {
+    // Everytime the route changes, check for auth status
+    '$route': 'isAuthenticated'
+  },
+  methods: {
+    async isAuthenticated () {
+      this.authenticated = await this.$auth.isAuthenticated()
+    },
+    async logout () {
+      await this.$auth.logout()
+
+      // Stay on current page and make sure it is refreshed
+      this.$router.push({ path: '/' })
+    }
+  }
+}
+</script>
+```
+
+### Use the Access Token
+When your users are authenticated, your React application has an access token that was issued by your Okta Authorization server. You can use this token to authenticate requests for resources on your server or API. As a hypothetical example, let's say you have an API that provides messages for a user. You could create a `MessageList` component that gets the access token and uses it to make an authenticated request to your server.
+
+Here is what the Vue component could look like for this hypothentical example using [axios](https://github.com/axios/axios):
+
+```typescript
+// src/components/MessageList.vue
+
+<template>
+  <ul v-if="posts && posts.length">
+    <li v-for="post in posts" :key='post.title'>
+      <p><strong>{{post.title}}</strong></p>
+      <p>{{post.body}}</p>
+    </li>
+  </ul>
+</template>
+
+<script>
+import axios from 'axios'
+
+export default {
+  data () {
+    return {
+      posts: []
+    }
+  },
+  async created () {
+    axios.defaults.headers.common['Authorization'] = `Bearer ${await this.$auth.getAccessToken()}`
+    try {
+      const response = await axios.get(`http://localhost:{serverPort}/api/messages`)
+      this.posts = response.data
+    } catch (e) {
+      console.error(`Errors! ${e}`)
+    }
+  }
+}
+</script>
+```
+
+### Reference
+**Configuration Options**
+  - `issuer` **(required)**: The OpenID Connect `issuer`
+  - `client_id` **(required)**: The OpenID Connect `client_id`
+  - `redirect_uri` **(required)**: Where the callback is hosted
+  - `scopes` *(optional)*: Reserved or custom claims to be returned in the tokens
+
+### Using a custom login-page
```

Comment:
Let's add a section that enumerates the available API methods, like what we have here: https://github.com/okta/okta-oidc-js/tree/master/packages/okta-react#auth